docs/slack.rst

```diff
@@ -27,16 +27,23 @@ Feel free to send a Pull Request to update `this page`_, if you want to include
 Slack History
 -------------
 
-We don't pay for our Slack, which means that the community has access to only the most
-recent 10,000 messages. Paying for Slack is prohibitively expensive for a community
-like ours, but many people seem to feel a freedom in knowing that their words will disappear shortly.
+We don't pay for our Slack, which means that the community has access to only the most recent 10,000 messages.
+Paying for Slack is prohibitively expensive for a community like ours,
+but many people seem to feel a freedom in knowing that their words will disappear shortly.
 We understand that not everyone loves this decision,
-but it's the way things are currently set up and we don't plan to change it in the near future. 
+but we don't plan to change it in the foreseeable future.
 
-That doesn't mean everything gets lost, though - you can always react to a message with the 
-``:suggest-for-newsletter:`` emoji to nominate it for inclusion in our next monthly newsletter. 
-Read the newsletter archives at :doc:`/newsletter` and subscribe if you want them directly in your inbox! 
+That doesn't mean everything gets lost, though.
```

Comment:
I was chewing on this - feel free to discard entirely - but I _think_ we can be more explicit about just how prohibitively expensive this is.

I also don't think we need to call out the pros vs cons (given that we're only talking about the commonly stated pro anyways). 

```suggestion
Paying for Slack is prohibitively expensive - for the size of our community (over 10K members), the lowest available paid Slack plan would cost somewhere in the range of 5-figures per month USD. 

Our desire to keep this community free-to-access means that the Write the Docs Slack will remain on the free plan. We have no plans to change or revisit this topic in the foreseeable future. 

There have been numerous discussions both internally and within the community on the upsides and downsides of a finite history - and we are certain there will be more in the future. We respect and welcome additional input and ideas. It's as a result of those discussions that we have discovered that a finite history does not mean that everything gets lost.
```

Reply:
I like what you're thinking here, Ravind, but I realized that adding more text here hides the actual thing we want to achieve: for the community to understand the limitation and to spread what they've learned beyond Slack.

I moved the "Why don't we pay for Slack?" question to it's own note box. That said, I didn't follow this suggestion fully because, while it's all true, I think there's some value in being assertive and not explaining ourselves too much.

Reply:
I think what you've both come up with looks really great! I think the "5 figures USD per month" thing is useful to highlight - a lot of people who ask us do so with the assumption that it can't be _that_ expensive, and a ballpark number might be helpful to head off those repeated inquiries.